Tools/AP_Bootloader/board_types.txt

```diff
@@ -514,6 +514,11 @@ AP_HW_AEROFOX_H7                     7110
 # IDs 7120-7129 reserved for Accton Godwit
 AP_HW_Accton_Godwit_GA1        7120
 
+# IDs 7130-7139 reserved for XRush4
```

Comment:
Consider making a separate PR just for this change - it's first in best dressed on these IDs.